app/templates/components/host-profile.hbs

```diff
@@ -92,19 +92,27 @@
 
       <h6>Favorite Food</h6>
       <p data-test-favorite-food>{{@profile.favoriteFood}}</p>
+    </div>
 
+    <div class="preview-section col-md-6">
       <h6>Movie Type</h6>
       <p data-test-movie-genre>{{movie-genre-display @profile.movieGenre}}</p>
-    </div>
-    <div class="preview-section col-md-6">
+
       <h6>Use Illicit Substance/Drug</h6>
       <p data-test-used-substances>{{substances-display @profile.mySubstances}}</p>
-
-      <h6>Accept Illicit Substance/Drug </h6>
-      <p data-test-acceptable-substances>{{substances-display @profile.residentSubstances}}</p>
     </div>
   </div>
 
+  <!-- To add backyard activities preview-section -->
```

Comment:
Just realized we have "backyard activities section" in the profile form but I did not add it into the profile preview page. Thus, I created a section here. Would be great to have your help set up the backend to get the data from the form to the profile preview page. Thanks!